modules/textures/src/lib/parsers/parse-compressed-texture.d.ts

```diff
@@ -0,0 +1,8 @@
+import { TextureLevel } from "@loaders.gl/textures/types/textrure";
```

Comment:
```suggestion
import { TextureLevel } from "@loaders.gl/textures/types/texture";
```

Typo `textrure` -> `texture` please search replace.